### ISSUE TITLE
I can't change camera2 orientation

### ISSUE THREAD

**Issue by developer--** [pedroSG94/rtmp-rtsp-stream-client-java#165]
for camera1 api there is no problem with changing camera orientation (0 90 180 270), but when I am using camera2 api 
`camera2.prepareVideo(640,480,30,1200 * 1024,true,90)` it does not work, I have tested with all degrees but nothing were changed at all, textureview preview is fine but the result stream is rotated

Lenovo (OS 6.0.0)

**Comment by pedroSG94** [comment]
If you are using surfaceview or textureview, hardware rotation only work for some device, if don't work this fail without error (it means that your encoder doesnt support it and encoder API doesn't throw error only ignore this param).
If you want rotate stream you can do it using lightopenglview or openglview, also remember set hardware rotate param to false to enable rotation with opengl.

**Comment by developer--** [comment]
But, my goal is to stream from service, is openglview relevant for that?

**Comment by pedroSG94** [comment]
Oh, if you want stream for service you can't do it with opengl for now. It is possible but I need develop it.
I will add it in future updates but now I'm in holidays so you will need wait.

**Comment by developer--** [comment]
Yes, I really need that :/ what I tried for openglview were I added view to the WindowManager with very small sizes and worked, but the result was stretched and very bad. 

Ok I'll wait for you, thanks at all really good work.

**Comment by pedroSG94** [comment]
This feature was added with camera1 and camera2: https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/commit/ce8514861c7c5b22ed9db9552932df8745f4ff07

**Comment by developer--** [comment]
Very nice and thank you, the only one strange behavior is that, when I am rotating device, frames also are rotating. Is there any way to prevent frame rotation?

**Comment by pedroSG94** [comment]
That rotation happens with all stream modes. It is like when you record a video with your camera app.
If you want change it you can do your own filter to change it. You need know Opengl.
https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/wiki/Real-time-filters

**Comment by developer--** [comment]
Do you plan to add filter like that? (I don't have any experience with Opengl)
